### PR TITLE
Issue 245/cm code quality

### PR DESCRIPTION
**Pull request by godanny86**
## Description

Fixes for several warnings triggered in Cloud Manager:

Package | Line Number | Issue
-- | -- | --
com.adobe.aem.guides:aem-guides-wknd.ui.apps:0.3.1-SNAPSHOT | 0 | Node /apps/wknd/components/image/cq:editConfig/cq:inplaceEditing/config   is an OSGi config or install path which contains non-OSGi-related children   that will not be visible to non-administrative users.
com.adobe.aem.guides:aem-guides-wknd.ui.config:0.3.1-SNAPSHOT | 0 | affected path /apps/wknd/osgiconfig overlaps   [com.adobe.aem.guides:aem-guides-wknd.ui.apps:0.3.1-SNAPSHOT]
com.adobe.aem.guides:aem-guides-wknd.core:src/main/java/com/adobe/aem/guides/wknd/core/models/HelloWorldModel.java | 30 | The class or interface org.apache.sling.settings.SlingSettingsService has   been deprecated and is not recommended to be used on AEM.
com.adobe.aem.guides:aem-guides-wknd.core:src/main/java/com/adobe/aem/guides/wknd/core/models/HelloWorldModel.java | 45 | Remove this use of "SlingSettingsService"; it is deprecated.

## Related Issue

Originally filed as issue #245 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: core/src/main/java/com/adobe/aem/guides/wknd/core/models/HelloWorldModel.java [hunk 1]
```diff
@@ -41,8 +39,6 @@
     @Default(values="No resourceType")
     protected String resourceType;
 
-    @OSGiService
-    private SlingSettingsService settings;
```
Comment (davidjgonzalez) [inline core/src/main/java/com/adobe/aem/guides/wknd/core/models/HelloWorldModel.java:45]: 👏

File: ui.apps/src/main/content/META-INF/vault/filter.xml [hunk 2]
```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
-    <filter root="/apps/wknd"/>
+    <filter root="/apps/wknd/clientlibs"/>
```
Comment (davidjgonzalez) [inline ui.apps/src/main/content/META-INF/vault/filter.xml:3]: What's the violation of using `/apps/wknd` ? 

`/apps/wknd` only has these three sub-paths, so what the advantage of enumerating each?

**Review by davidjgonzalez — APPROVED** [review]
I trust the finer grained filter.xml enumeration is for good reason, so approving (curious to hear what it is though).

File: ui.apps/src/main/content/META-INF/vault/filter.xml (hunk 2 above)
Comment (godanny86) [reply ui.apps/src/main/content/META-INF/vault/filter.xml:3]: it explicitly avoid a potential overlap with the `ui.config` module which contains the path `/apps/wknd/osgiconfig`

**Pull request merged**